src/postgres/commands/createPostgresServer/steps/PostgresServerCreateStep.ts

```diff
@@ -30,11 +31,11 @@ export class PostgresServerCreateStep extends AzureWizardExecuteStep<IPostgresSe
                 const options = {
                     location: locationName,
                     sku: {
-                        name: "GP_Gen5_4",
-                        tier: "GeneralPurpose",
-                        capacity: 4,
+                        name: "B_Gen5_1",
+                        tier: "Basic",
+                        capacity: 1,
                         family: "Gen5",
-                        size: "102400"
+                        size: `postgresDefaultStorageSizeMB`
```

Comment:
Does this work? I normally just do `String(storageSizeMB)`. If you use template literal, I thought you would need `${`

Reply:
That's true, I forgot to put the ${. I tested it and it was working so didn't realize, maybe the value was pulled from the second variable instead of this.